core/src/main/kotlin/live/LiveMessage.kt

```diff
@@ -162,6 +163,8 @@ public class LiveMessage(
         is ChannelDeleteEvent -> event.channel.id == message.channelId
 
         is GuildDeleteEvent -> event.guildId == guildId
+
+        is InteractionCreateEvent -> event.interaction.data.message.value?.id == message.id
```

Comment:
The event might also be related to the message if a message command was invoked on it (`MessageCommandInteractionCreateEvent`).

Reply:
But that doesn't affect it's data

Reply:
Hm  true, I'm not so familiar with Live Entities yet to be honest.